test/Maps/MapReader.test.cpp

```diff
@@ -1,19 +1,17 @@
 #include "Maps/MapReader.h"
-#include "Maps/MapType.h"
 #include <gtest/gtest.h>
 #include <string>
 
-TEST(MapReader, EmptyFilename) {
-	EXPECT_THROW(MapReader::Read(std::string(""), MapType::Map), std::runtime_error);
-	EXPECT_THROW(MapReader::Read(std::string(""), MapType::Save), std::runtime_error);
-}
-
 TEST(MapReader, MissingFile) {
-	EXPECT_THROW(MapReader::Read("MissingFile.map", MapType::Map), std::runtime_error);
-	EXPECT_THROW(MapReader::Read("MissingFile.op2", MapType::Save), std::runtime_error);
+	EXPECT_THROW(MapReader::ReadMap("MissingFile.map"), std::runtime_error);
+	EXPECT_THROW(MapReader::ReadSavedGame("MissingFile.op2"), std::runtime_error);
+
+	// Check if filename is an empty string
+	EXPECT_THROW(MapReader::ReadMap(std::string("")), std::runtime_error);
+	EXPECT_THROW(MapReader::ReadSavedGame(std::string("")), std::runtime_error);
```

Comment:
To clear up any confusion concerning these lines, my intention was to remove the explicit `std::string` wrapper:
```cpp
EXPECT_THROW(MapReader::ReadMap(""), std::runtime_error);
EXPECT_THROW(MapReader::ReadSavedGame(""), std::runtime_error);
```

Reply:
oh, okay